scripts/seed.py

```diff
@@ -24,6 +24,12 @@
 
 
 def create_user(is_admin, username=""):
+    """
+    Creates superuser, participant user, host user and returns it.
+
+    To create a user, we must enter username, email and password of the user.
```

Comment:
The doc string must tell what a function does in as less words as possible. If we start writing comments that describe what the code is doing, then that is not neat. What say @deshraj